src/openrct2/paint/Supports.cpp

```diff
@@ -473,7 +551,7 @@ bool WoodenASupportsPaintSetup(
         imageTemplate = ImageId().WithTransparency(FilterPaletteID::PaletteDarken1);
     }
 
-    auto baseHeight = Floor2(session.Support.height + 15, 16);
+    uint16_t baseHeight = Floor2(session.Support.height + 15, 16);
```

Comment:
Seems like this changed is tied to what happened on commit 1 and should be moved there.